build.gradle.kts

```diff
@@ -121,7 +121,7 @@ task<Test>("documentGenerationTest") {
 
 java {
   toolchain {
-    languageVersion.set(JavaLanguageVersion.of(16))
+    languageVersion.set(JavaLanguageVersion.of(16)) // TODO Need Kotlin 1.6 to target JVM 17 - see PUD-1241
```

Comment:
cant we use jdk 17 here, but just accept the warnings?

Reply:
As per comment, we'll get there when we go to Kotlin 1.6.